MapboxCoreNavigation/RouteController.swift

```diff
@@ -425,7 +425,9 @@ extension RouteController: CLLocationManagerDelegate {
     func updateRouteLegProgress(for location: CLLocation) {
         let currentDestination = self.routeProgress.currentLeg.destination
 
-        guard self.routeProgress.currentLegProgress.remainingSteps.count <= 1 else { return }
+        guard self.routeProgress.currentLegProgress.remainingSteps.count == 0 else {
```

Comment:
We should not do this :), can we use distance or time remaining? For this to become true, the arrival needs to be exactly triggered, which is especially an issue for multi leg routes.

Reply:
Ah, sorry, I just saw your comment. To be honest, I would prefer if we could actually create another variable like the RouteControllerManeuverZoneRadius, but for leg progress distance.

Reply:
`RouteControllerManeuverZoneRadius` is used 5 times in the code base, and it seems to be used pretty consistently to say:

> How close do we need to be to a thing to be considered "at" that thing?

TBH, that seems exactly in line with how we're using it here. Why would we want it to be different ? Do you actually have this use case? What value would you choose for it?

Examples pulled from the code:
---

Are we "at" the beginning of a leg:
```
let isWithinDepartureStep = distanceToFirstCoordinateOnLeg < RouteControllerManeuverZoneRadius
```

Are we "at" the intersection:
```
if absoluteDistanceToIntersection <= RouteControllerManeuverZoneRadius {
```

Are we at/along the current step?
```
// If user is close to an intersection, the radius will be lower, so reroutes will fire easier.
let radius = max(reroutingTolerance, RouteControllerManeuverZoneRadius)
```

Are we "at" the end of a step?
```
if userSnapToStepDistanceFromManeuver <= RouteControllerManeuverZoneRadius,
```

Are we *not* "at" the start of the route?
```
lastKnownUserAbsoluteDistance > RouteControllerManeuverZoneRadius
```